### ISSUE TITLE
Crash in usdview when displaying an instance using "model:drawMode=bounds"

### ISSUE THREAD

**Issue by fabal** [PixarAnimationStudios/USD#1252]
### Description of Issue
usdview seems to crash in USD 20.05 under certain circumstances.

### Steps to Reproduce
1. Load this file in usdview

```
#usda 1.0

def "Ball" (
    instanceable = true
    kind = "component"
    # "Ball.geom.usd" is the asset used in the Kitchen set
    prepend references = @./Ball.geom.usd@
)
{
    uniform token model:drawMode = "bounds"
}

```

Here's the error log

```
------------------------ 'python3.7' is dying ------------------------
python3.7 crashed. FATAL ERROR: Used null prim
in Usd_IssueFatalPrimAccessError at line 426 of ../../../pxr/usd/usd/primData.cpp

Thread 139647923038016 Pending Diagnostics:
Coding Error: in _GetPrimPathFromInstancerChain at line 537 of ../../../pxr/usdImaging/usdImaging/primAdapter.cpp -- Failed verification: ' prim.IsInMaster() '
```

### System Information (OS, Hardware)
Linux
### Package Versions
USD 20.05

**Comment by jtran56** [comment]
Filed as internal issue #USD-6172.

**Comment by tcauchois** [comment]
Hi,

Just to check: did this crash happen when you picked the bounds-mode prim?  That's the only way I was able to repro the crash.

If so, the fix is in dev and 20.08.

Cheers,
Tom

**Comment by fabal** [comment]
Thanks Tom. In our case it crashed straight after the display, without picking the prim but the good news is that it's fixed in 20.08 :+1:
